go.mod

```diff
@@ -2,29 +2,28 @@ module github.com/rancher/norman
 
 go 1.12
 
-replace github.com/matryer/moq => github.com/rancher/moq v0.0.0-20190404221404-ee5226d43009
+replace (
+	github.com/matryer/moq => github.com/rancher/moq v0.0.0-20190404221404-ee5226d43009
+	k8s.io/client-go => k8s.io/client-go v0.0.0-20190918160344-1fbdaa4c8d90
```

Comment:
why is this replace here?

Reply:
Since we don't want to update wrangler, client-go gets incompatible `1a26190bd76a` from wrangler. Thus adding k8s.io/client-go in replace with required version.